Jenkinsfile

```diff
@@ -24,6 +25,7 @@ pipeline {
                 syncOpenShiftSecret params.STAGE_SECRET_NAME
             }
         }
+        /*
```

Comment:
Why comment out the testing stage? I guess this is to speed up testing? :wink:

Reply:
@codificat yup especially since I have to run a "few" times to get it working right.